### PR TITLE
Adding a README at the top repository to explain about the setup of configuration

### PR DESCRIPTION
**Pull request by laudares**
Hi @gkristic, we've discussed this need in a [conversation](https://app.intercom.io/a/apps/704c9e2a2a2877207ecc28c1990b0342756f9f35/respond/inbox/conversation/10364189653) with @JukeI3ug, @potocnyj and @seaparks .

I've looked at this issue through different angles and in my opinion creating this README file at the top of our VirtualCortex/docker repository would be a fitting solution, giving we redirect users from our [Containerized Installation](https://docs.vividcortex.com/getting-started/containerized-installation/) documentation page straight here.

### CODE REVIEW

**Comment by laudares** [comment]
Thank you for the review and suggestions, @gkristic - I've implemented both of them.

**Comment by laudares** [comment]
@seaparks - do you have enough privileges to merge this branch ?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Please have in mind that configuration for containers is controlled via environment variables that should be set in one of the following ways:
+1. as build arguments when building the image
+2.  passed as configuration parameters at runtime
```
Comment (Preetam) [inline README.md]: extra space

**Comment by laudares** [comment]
Strange, it reads:
> Add more commits by pushing to the master branch on laudares/docker.
I've just done that, removing the extra space pointed by @Preetam, but even though the change shows up in https://github.com/laudares/docker it wasn't pushed here. Does anyone knows why ?

**Comment by seaparks** [comment]
nope I don't have the perms for this repo. @Preetam may though

**Comment by Preetam** [comment]
Eh, nbd about the space. I'll merge.

**Comment by laudares** [comment]
Thank you, @Preetam !

**Pull request merged**